server/controllers/routes/admin/cohort/deleteCohort.js

```diff
@@ -0,0 +1,21 @@
+const { deleteCohortQuery } = require('../../../../database/queries');
+
+const deleteCohort = async (req, res, next) => {
+  try {
+    const check = await deleteCohortQuery(req.params.cohortId);
```

Comment:
use destructuring here:
``` const { cohortid } = req.params;```
```suggestion
    const check = await deleteCohortQuery(cohortId);
```